components/scream/src/physics/cld_fraction/atmosphere_cld_fraction.cpp

```diff
@@ -55,9 +55,9 @@ void CldFraction::set_grids(const std::shared_ptr<const GridsManager> grids_mana
 void CldFraction::initialize_impl (const RunType /* run_type */)
 {
   // Set property checks for fields in this process
-  auto frac_interval_check = std::make_shared<FieldWithinIntervalCheck>(0,1);
-  add_property_check<Computed>(get_field_out("cldfrac_ice").get_header().get_identifier(),frac_interval_check);
-  add_property_check<Computed>(get_field_out("cldfrac_tot").get_header().get_identifier(),frac_interval_check);
+  using FWIC = FieldWithinIntervalCheck;
+  add_post_run_property_check<FWIC>(get_field_out("cldfrac_ice"),0,1);
```

Comment:
I like this economical syntax, which feeds the arguments of `add_x_property_check` to the specified property check constructor. Just curious: do you think anyone who uses it might find it confusing, as opposed to passing an instance as the second argument instead?

Reply:
I am not sure. I provided both constructors, so people can decide for themselves. I like the economy version too, so I use that when I'm the one coding. But I guess the two steps version leaves less room for interpretation, so some might like that.